Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -11660,9 +11660,8 @@ void WebPageProxy::Internals::setPlaybackTarget(PlaybackTargetClientContextIdent
         return;
 
     auto context = target->targetContext();
-    ASSERT(context.type() != MediaPlaybackTargetContext::Type::SerializedAVOutputContext);
     if (page.preferences().useGPUProcessForMediaEnabled())
-        context.serializeOutputContext();
+        context.outputContext();
```

Comment:
What are you trying to achieve here? `outputContext()` looks like a trivial getter and you're not using the value it returns. This looks like a no-op?